smarts/sstudio/types.py

```diff
@@ -559,9 +559,41 @@ class _ActorAndMission:
     mission: Union[Mission, EndlessMission, LapMission]
 
 
+@dataclass(frozen=True)
+class VehicleType:
```

Comment:
How do you imagine the use of this this vehicle is distinguished from the use of traffic actors in scenario studio?

Reply:
Yes, they look similar because they are both leaking the SUMO's implementation details. For now, `VehicleType` is transparent to SMARTS, which means SMARTS just pass everything to the underlying implementation.

Reply:
OK, The duplication requires another pass on the scenario studio types, I think this can go in with another issue raised addressing these noted problems.